restapi/src/main/java/io/stargate/web/docsapi/resources/DocumentResourceV2.java

```diff
@@ -238,21 +388,57 @@ public Response deleteDocPath(
   }
 
   @GET
+  @ApiOperation(
+      value = "Get a document",
+      notes = "Retrieve the JSON representation of the document",
+      response = ResponseWrapper.class)
+  @ApiResponses(
+      value = {
+        @ApiResponse(code = 200, message = "OK", response = ResponseWrapper.class),
+        @ApiResponse(code = 204, message = "No Content"),
+        @ApiResponse(code = 400, message = "Bad Request"),
+        @ApiResponse(code = 401, message = "Unauthorized"),
+        @ApiResponse(code = 403, message = "Forbidden"),
+        @ApiResponse(code = 500, message = "Internal Server Error", response = Error.class)
+      })
   @Path("{namespace-id: [a-zA-Z_0-9]+}/collections/{collection-id: [a-zA-Z_0-9]+}/{document-id}")
   @Consumes("application/json")
   @Produces("application/json")
   public Response getDoc(
       @Context HttpHeaders headers,
       @Context UriInfo ui,
-      @HeaderParam("X-Cassandra-Token") String authToken,
-      @PathParam("namespace-id") String namespace,
-      @PathParam("collection-id") String collection,
-      @PathParam("document-id") String id,
-      @QueryParam("where") String where,
-      @QueryParam("fields") String fields,
-      @QueryParam("page-size") int pageSizeParam,
-      @QueryParam("page-state") String pageStateParam,
-      @QueryParam("raw") Boolean raw) {
+      @ApiParam(
+              value =
+                  "The token returned from the authorization endpoint. Use this token in each request.",
+              required = true)
+          @HeaderParam("X-Cassandra-Token")
+          String authToken,
+      @ApiParam(value = "the namespace that the collection is in", required = true)
+          @PathParam("namespace-id")
+          String namespace,
+      @ApiParam(value = "the name of the collection", required = true) @PathParam("collection-id")
+          String collection,
+      @ApiParam(value = "the name of the document", required = true) @PathParam("document-id")
+          String id,
+      @ApiParam(value = "a JSON blob with the search filters", required = false)
+          @QueryParam("where")
+          String where,
```

Comment:
I'm wondering if we shouldn't make stuff like "where" a concrete type like an enum so that it's easier to see the possible values. It may make documentation easier too.

Reply:
Other than Swagger, is there any supplemental FAQ/getting started material that we are publishing? Because if swagger is the only source of docs, I think I need to make this more explanatory